playbooks/roles/openvpn/tasks/main.yml

```diff
@@ -1,4 +1,15 @@
 ---
+- name: Import the public GPG key from OpenVPN community
+  apt_key: url=https://swupdate.openvpn.net/repos/repo-public.gpg state=present
+
+- name: Adds the OpenVPN community repositroy to sources.list
+  apt_repository: repo='deb http://swupdate.openvpn.net/apt trusty main' state=present
+
+- name: Ensure the APT cache is up to date
```

Comment:
This task isn't necessary. The `apt_repository` module automatically refreshes the cache when a new repository is added.